### PR TITLE
Updates the description of rechargers

### PR DESCRIPTION
**Pull request by Archimus12**
## About The Pull Request
Changes the description of chargers to mention their ability to charge PDAs.

## Why It's Good For The Game
Makes it more clear how to charge PDAs so people know they can be recharged instead of having to have their cell replaced.
Also makes the rechargers description more accurate.

## Changelog
:cl:
spellcheck: Makes the description of rechargers mention that they can recharge PDAs as well.
/:cl:

### CODE REVIEW

File: code/game/machinery/recharger.dm [hunk 1]
```diff
@@ -3,7 +3,7 @@
 	icon = 'icons/obj/stationobjs.dmi'
 	icon_state = "recharger"
 	base_icon_state = "recharger"
-	desc = "A charging dock for energy based weaponry."
+	desc = "A charging dock for energy based weaponry and PDAs."
```
Comment (msgerbs) [inline code/game/machinery/recharger.dm:6]:
```suggestion
	desc = "A charging dock for energy based weaponry, PDAs, and other devices."
```

Maybe just kill 2 birds with one stone here? They can charge the tagger I believe and there's probably others as well.

Comment (Archimus12) [reply code/game/machinery/recharger.dm:6]: What tagger do they recharge?

Comment (Archimus12) [reply code/game/machinery/recharger.dm:6]: it charges PDAs, tablets, batons, energy weapons and energy magazines so sure

Comment (msgerbs) [reply code/game/machinery/recharger.dm]: Huh, I could've sworn destination taggers used cells and needed to be recharged. Shows how much I play cargo.

**Review by msgerbs — APPROVED** [review]
Works for me

**Comment by Jackal-boop** [comment]
This PR should chance the microwave examine text to indicate you can also charge your PDA in it

**Comment by optimumtact** [comment]
Welcome aboard

**Comment by Archimus12** [comment]
Thanks

**Pull request merged**